model_compression_toolkit/target_platform_capabilities/target_platform/target_platform_model.py

```diff
@@ -96,7 +96,7 @@ def get_config_options_by_operators_set(self,
         for op_set in self.operator_set:
             if operators_set_name == op_set.name:
                 return op_set.qc_options
-        return get_default_quantization_config_options()
+        return self.default_qco
```

Comment:
anyone else use get_default_quantization_config_options()? or you can remove it?